trajectory_tracker_msgs/CMakeLists.txt

```diff
@@ -36,8 +36,8 @@ if(CATKIN_ENABLE_TESTING)
 
   add_compile_options(-std=c++11)
 
-  catkin_add_gtest(test_convert test/src/test_convert.cpp)
-  target_link_libraries(test_convert ${catkin_LIBRARIES} ${CMAKE_THREAD_LIBS_INIT})
+  catkin_add_gtest(test_conversion test/src/test_conversion.cpp)
```

Comment:
Something like `test_path_with_velocity_conversion` would be better to avoid any other naming collision.